ci/dev.yaml

```diff
@@ -0,0 +1,18 @@
+name: xvec_dev
+channels:
+  - conda-forge
+  - conda-forge/label/shapely_dev
+dependencies:
+  - python
+  # to build from source
+  - cython
+  - geos
+  # testing
+  - pytest
+  - pytest-cov
+  - pytest-xdist
+  - pytest-reportlog
+  - pip
+  - pip:
+      - git+https://github.com/shapely/shapely.git@main
```

Comment:
nothing we can do about here, but I wonder if rather than building it ourselves `shapely` could upload "nightly wheels" somewhere once per day (to TestPyPI / [scipy-wheels-nightly](https://anaconda.org/scipy-wheels-nightly/) / somewhere else)? If there's lots of projects running upstream-dev CI (and I believe every project *should* do that) those pre-built binaries would save quite a bit of resources

Reply:
@jorisvandenbossche was there any discussion on shapely nightly builds? Any idea how many other projects would be interested in that apart from geopadnas and xvec?